### PR TITLE
Bank Account Model

### PR DESCRIPTION
**Pull request by bibiferreira**
added model for bank account and connected it to user and user_cues + updated the seed

### CODE REVIEW

File: app/models/account.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Account < ApplicationRecord
+  belongs_to :user
+
+  has_many :creditor_user_cues, class_name: "UserCue", inverse_of: :creditor_account
```
Comment (yestephan) [inline app/models/account.rb:4]: @bibiferreira ah this is complicated, I understand the problem now.

Comment (yestephan) [inline app/models/account.rb:4]: @bibiferreira :creditor_user_cues?

File: app/models/user_cue.rb [hunk 2]
```diff
@@ -1,4 +1,7 @@
 class UserCue < ApplicationRecord
   belongs_to :user
```
Comment (yestephan) [inline app/models/user_cue.rb:2]: belongs to user through accounts instead?

Comment (bibiferreira) [reply app/models/user_cue.rb:2]: if we do this then if you delete an account you would lose all cues you have set for that account + this way it seems faster to query all cues from user and the other way around as well

File: app/models/account.rb (hunk 1 above)
Comment (bibiferreira) [reply app/models/account.rb:4]: this is just an extra, not really mandatory but in this way we could list all the cues associated to an account

File: app/models/user_cue.rb (hunk 2 above)
Comment (yestephan) [reply app/models/user_cue.rb:2]: Cool!

File: app/models/account.rb (hunk 1 above)
Comment (yestephan) [reply app/models/account.rb:4]: Ok gotcha, nice

**Pull request merged**